lib/src/test/java/space/taran/arklib/TestingUtils.kt

```diff
@@ -0,0 +1,25 @@
+package space.taran.arklib
+
+import dev.arkbuilders.arklib.ResourceId
+import dev.arkbuilders.arklib.user.score.Score
+
+fun randomResourceId(): ResourceId =
+    ResourceId(
+        (Math.random() * 1000).toLong() + 1,
+        (Math.random() * 1000).toLong(),
+    )
+
+fun randomScore(max: Int = 1000): Score =
+    (Math.random() * (max - 1)).toInt() + 1
+
+fun randomPositiveInt(max: Int = 1000): Int {
+    if (max < 10) {
+        throw Exception("parameter max value must be greater than equal to 10")
+    }
```

Comment:
Why?

Reply:
@kirillt a max value >= 10 ensure a probability of 90% + that the number is non zero.

Math.random returns a double value between 0 and 1. If max value is 1 then the probability of getting 0 is 99%, if it is 2 then the probability is 50% and so on.

Reply:
You don't need `Math.random`. For eliminating 0, it's only needed to use lower bound, e.g. `Random.nextLong(1, <higher bound>)`